src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int AMOUNT_INDEX = 1;
+    private static final String OPERATION_TYPE_BUY = "buy";
+    private static final String OPERATION_TYPE_SUPPLY = "supply";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        writeStatistic(toFileName, getOperationType(readStatistic(fromFileName)));
+    }
+
+    public String[] readStatistic(String fromFileName) {
+        File fileReade = new File(fromFileName);
+        StringBuilder stringBuilder;
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(fileReade));
+            stringBuilder = new StringBuilder();
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                stringBuilder.append(value).append(System.lineSeparator());
+                value = bufferedReader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file", e);
```

Comment:
```suggestion
            throw new RuntimeException("Can't read file data from file " +  fromFileName, e);
```

lets make exception messages more informative